dojo/product/views.py

```diff
@@ -1052,7 +1052,11 @@ def edit_technology(request, tid):
                                  'Technology changed successfully.',
                                  extra_tags='alert-success')
             return HttpResponseRedirect(reverse('view_product', args=(technology.product.id,)))
+        else:
+            logger.debug('delete_product: POST INVALID FORM')
+            logger.error(form.errors)
```

Comment:
just having an error on a form does not warrant the logging at the highest severity level. should be debug.